app/routes/groups_.new.tsx

```diff
@@ -1,11 +1,11 @@
-import { Card } from '~/components/ui/containers';
+import { Card, DashboardHeader } from '~/components/ui/containers';
 
 export default function GroupNew() {
   return (
     <div className="grow py-20 lg:py-40 bg-secondary w-full flex flex-col items-center justify-center">
       <div className="flex w-full items-center justify-center flex-col">
         <div className="max-w-[900px] w-full flex flex-col gap-3">
-          <h1 className="text-black font-bold text-4xl">Create New Group</h1>
+          <DashboardHeader>Create New Group</DashboardHeader>
```

Comment:
NIT: Maybe just `H1`? `DashboardHeader` also works, though!

Reply:
That'd be easy, is it common to mirror common tags and use caps?

Reply:
Definitely seen it before! Mostly for `button` -> `Button` and headings `h1` -> `H1`. It's a nice pattern as the component name clearly communicates that it's a wrapper around an HTML element that adds some app-specific styling or functionality!